app/main.go

```diff
@@ -116,10 +139,11 @@ func cleanup(w http.ResponseWriter, r *http.Request) {
 		metricNames = append(metricNames, m.Name)
 	}
 
-	storageManager, err := loadStorageEngine()
+	storageManager, err := loadStorageEngine(ctx)
```

Comment:
Do you need a second copy of the storage engine here?

Reply:
Good point, no. Thank you!